packages/x-components/src/components/layouts/layout-single-column.vue

```diff
@@ -0,0 +1,218 @@
+        <span v-if="devMode" class="slot-helper">HEADER</span>
+      </slot>
+    </header>
+    <div v-if="hasContent('sub-header')" class="x-layout__sub-header x-list x-list--horizontal">
+      <!-- @slot Slot that is be used for insert content into the Sub Header. -->
+      <slot name="sub-header">
+        <span v-if="devMode" class="slot-helper">SUB HEADER</span>
+      </slot>
+    </div>
+
+    <div v-if="hasContent('toolbar')" class="x-layout__toolbar x-list x-list--horizontal">
+      <!-- @slot Slot that is be used for insert content into the Toolbar. -->
+      <slot name="toolbar">
+        <span v-if="devMode" class="slot-helper">TOOLBAR</span>
+      </slot>
+    </div>
+
+    <div v-if="hasContent('predictive')" class="x-layout__predictive x-list x-list--vertical">
+      <BaseScroll>
+        <!-- @slot Slot that is be used for insert content into the Predictive Layer. -->
+        <slot name="predictive">
+          <span v-if="devMode" class="slot-helper">PREDICTIVE</span>
+        </slot>
+      </BaseScroll>
+    </div>
+
+    <main v-if="hasContent('main')" class="x-layout__main x-list x-list--vertical">
+      <BaseIdScroll id="main-scroll">
+        <!-- @slot Slot that is be used for insert content into the Main. -->
+        <slot name="main">
+          <span v-if="devMode" class="slot-helper">MAIN</span>
+        </slot>
+      </BaseIdScroll>
+    </main>
+
+    <div v-if="hasContent('floating')" class="x-layout__floating x-list x-list--horizontal">
+      <!-- @slot Slot that is be used for insert content into the Floating. -->
+      <slot name="floating">
+        <span v-if="devMode" class="slot-helper">FLOATING</span>
+      </slot>
+    </div>
+
+    <footer v-if="hasContent('footer')" class="x-layout__footer x-list x-list--horizontal">
+      <!-- @slot Slot that is be used for insert content into the Footer. -->
+      <slot name="footer">
+        <span v-if="devMode" class="slot-helper">FOOTER</span>
+      </slot>
+    </footer>
+
+    <BaseIdModal
+      v-if="hasContent('aside')"
+      :animation="asideAnimation"
+      modalId="aside-modal"
+      class="x-layout__aside"
+    >
+      <!-- @slot Slot that is be used for insert content into the Aside. -->
+      <slot name="aside">
+        <span v-if="devMode" class="slot-helper">ASIDE</span>
+      </slot>
+    </BaseIdModal>
+  </div>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue';
+  import { Component, Prop } from 'vue-property-decorator';
+  import BaseIdModal from '../modals/base-id-modal.vue';
+  import TranslateFromRight from '../animations/translate-from-right.vue';
+  import BaseIdScroll from '../scroll/base-id-scroll.vue';
+  import BaseScroll from '../scroll/base-scroll.vue';
+
+  /**
+   * Component for use as Layout to be filled with the rest of the Components.
+   *
+   * @public
+   */
+  @Component({
+    components: { BaseIdScroll, BaseScroll, BaseIdModal }
+  })
+  export default class LayoutS extends Vue {
```

Comment:
It's not because Spanish but because tokens naming:  (`component-variant`). But I guess we can change the name of the component and keep the tokens.

Reply:
I'm not sure about HolyGrail. Because we will have more "HolyGrailish" layouts. The jumbo layout is also a holy grail with the asides overlapping.

I want to leave clear what this layout does. And it is positioning elements following a multicolumn, and also with maxWidth.

Maybe `MulticolumnMaxwidthLayout` ?